websites/W/Wolvesville/presence.ts

```diff
@@ -0,0 +1,266 @@
+    showMenuState: boolean = await presence.getSetting("menuDetail"),
+    showPlayerCount: boolean = await presence.getSetting("playerCount"),
+    logo: number = await presence.getSetting("logo"),
+    logoArr = ["wov", "wov_no_bg", "wov_tile", "wov_web", "wov_blue", "wov_red", "wov_orange", "wov_yellow", "wov_green", "wov_purple", "wov_pink", "wov_white"],
+
+    data: PresenceData = {
+      largeImageKey: logoArr[logo] || "wov",
+      startTimestamp: startedTime
+    };
+
+  if (document.location.href.includes("blog.wolvesville.com")) { //Wolvesville Blog
+    data.smallImageKey = "wov_blog";
+    data.smallImageText = "Development Blog";
+    if (document.location.pathname === "/" || document.location.pathname.startsWith("/page")) {
+      data.details = "Development Blog";
+      data.state = "Browing posts...";
+    } else {
+
+      const post = document.querySelector(
+        ".post-title"
+      )
+        ? true : false;
+
+      if (post) {
+        data.details = "Reading a blog post:";
+        data.state = document.querySelector(".post-title").textContent;
+        data.buttons = [
+          {
+            label: "Read Post",
+            url: document.URL
+          }
+        ];
+      }
+    }
+  } else if (document.location.href.includes("legal.wolvesville.com")) { //Legal
+    data.details = "Legal";
+    if (document.location.pathname.includes("tos"))
+      data.state = "Reading the Terms of Service";
+    else if (document.location.pathname.includes("privacy-policy"))
+      data.state = "Reading the Privacy Policy";
+    else if (document.location.pathname.includes("imprint"))
+      data.state = "Reading the imprint";
+
+  } else if (document.location.href.includes("heroes.wolvesville.com")) { //Wolvesvile Heroes
+    data.details = "Wolvesville Heroes";
+    data.state = "In the home page";
+    data.smallImageKey = "wov_heroes";
+    data.smallImageText = "Wolvesville Heroes";
+
+    if (document.location.pathname.includes("overview"))
+      data.state = "Viewing Overview";
+    else if (document.location.pathname.includes("applications"))
+      data.state = "Viewing Applications";
+    else if (document.location.pathname.includes("updates"))
+      data.state = "Viewing Updates";
+    else if (document.location.href.includes("list?role") || document.location.href.includes("list.html?role")) {
+      const role = document.querySelector("#staff_member_name").textContent;
+      data.state = `Reading about the ${role} role`;
+    } else if (document.location.href.includes("list?member") || document.location.href.includes("list.html?member")) {
+      const member = document.querySelector("#staff_member_name").textContent;
+      data.state = `Reading about ${member}`;
+    }
+  } else if (document.location.href.includes("voting.wolvesville.com")) { //Voting Gallery
+    data.details = "Voting Gallery";
+
+    const submissionView = document.querySelector(
+      ".css-757v71"
+    )
+      ? true : false;
```

Comment:
You could do something like this to make it more clean
```js
    const submissionView = !!document.querySelector(
      ".css-757v71"
    );
```

Reply:
oh great, thanks